src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltoken/InputIllegalTokensCheckCommentsContent.java

```diff
@@ -1,16 +1,16 @@
-/* // violation, 'Using ' \/\/ violation\\nIllegalToken\\ntokens = COMMENT_CONTENT\\n\\n\\n' is not allowed'
+/* // violation
 IllegalToken
 tokens = COMMENT_CONTENT
```

Comment:
header comment should be just a config and nothing else.
you already moved this vioaltion comment under, so on line 1 it should be just removed.

Reply:
https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#COMMENT_CONTENT 
this is challange , as all comment will be violated.